charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/configmap.yaml

```diff
@@ -8,17 +8,15 @@ metadata:
     role: logging
 data:
   elasticsearch.yml: |-
-    node:
-      master: ${NODE_MASTER:true}
-      name: ${NODE_NAME:${HOSTNAME}}
-      data: ${NODE_DATA:true}
-      ingest: ${NODE_INGEST:true}
-      max_local_storage_nodes: ${MAX_LOCAL_STORAGE_NODES:1}
+    node.master: ${NODE_MASTER:true}
+    node.name: ${NODE_NAME:${HOSTNAME}}
+    node.data: ${NODE_DATA:true}
+    node.ingest: ${NODE_INGEST:true}
+    node.max_local_storage_nodes: ${MAX_LOCAL_STORAGE_NODES:1}
```

Comment:
I would personally prefer the old way. Is there a reason to switch the notation from nested to flat one? You could revert except the searchguard addition.